### PR TITLE
Introduction240

### PR DESCRIPTION
**Pull request by lindafoodev**
Added action, reducer, react-redux intro page component, edit app.js and store.js files to include introduction component/router and introReducer, respectively

Takes into consideration condition where user:
already viewed intro => setup to redirect (as discussed, not working pending you/scott futureconnection)
clicks after viewing final card => setup to redirect (as discussed, not working pending you/scott future connection)
views first intro card on mount component => shows appropriate first card
click for next card => shows appropriate next card

### CODE REVIEW

File: src/reducers/intro-reducer.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+} from "../actions/intro-page";
+
+const initialState = {
+  introCard: [
+    {number: 0, title: 'So You Want to Be An Investor...', details: 'There are a couple of things you should know about to get started.'},
+    {number: 1, title: 'Stocks', details: 'Stocks Lorem Ipsum Concise'},
+    {number: 2, title: 'Bonds', details: 'Bonds Lorem Ipsum Concise'},
+    {number: 3, title: 'Managed Fund', details: 'Managed Fund Lorem Ipsum Concise'}],
+  numCard: 0,
+  introComplete: true
```
Comment (vampaynani) [inline src/reducers/intro-reducer.js]: Intro complete needs to be `true` at the beginning?

**Review by vampaynani — COMMENTED** [review]
Store has conflicts with its master version, fix it before merge.
Move the return <Redirect .../> to the render method instead of being part of componentDidMount and nextCardClick. 
```
render(){
if(this.props.numCard === 3 || this.props.introComplete) return <Redirect to="/dashboard" />
...
}
```

File: src/reducers/intro-reducer.js (hunk 1 above)
Comment (lindafoodev) [reply src/reducers/intro-reducer.js]: Good catch - needs to default false at beginning, i had changed it to test the user condition - user already completed intro. Thanks!

Comment (lindafoodev) [reply src/reducers/intro-reducer.js]: Made changes to file

**Comment by ASTribble** [comment]
The conflicts in the store were with making sure we wanted to add  another new reducer.  
Linda and I looked at them together, didn't commit the changes, so I re-implemented those changes.

**Pull request merged**